tests/manage/pv_services/pvc_resize/test_pvc_expansion.py

```diff
@@ -0,0 +1,212 @@
+import logging
+import pytest
+from concurrent.futures import ThreadPoolExecutor
+
+from ocs_ci.ocs import constants
+from ocs_ci.utility.utils import TimeoutSampler
+from ocs_ci.framework.testlib import skipif_ocs_version, ManageTest, tier1
+from tests import helpers
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@skipif_ocs_version('<4.5')
+@pytest.mark.polarion_id('OCS-302')
+class TestPvcExpand(ManageTest):
+    """
+    Tests to verify PVC expansion
+
+    """
+    @pytest.fixture(autouse=True)
+    def setup(self, multi_pvc_factory, pod_factory):
+        """
+        Create resources for the test
+
+        """
+        access_modes_cephfs = [
+            constants.ACCESS_MODE_RWO, constants.ACCESS_MODE_RWX
+        ]
+        access_modes_rbd = [
+            constants.ACCESS_MODE_RWO, f'{constants.ACCESS_MODE_RWO}-Block',
+            f'{constants.ACCESS_MODE_RWX}-Block'
+        ]
+
+        self.pvcs_cephfs = multi_pvc_factory(
+            interface=constants.CEPHFILESYSTEM, size=10,
+            access_modes=access_modes_cephfs, status=constants.STATUS_BOUND,
+            num_of_pvc=2, timeout=90
+        )
+
+        self.pvcs_rbd = multi_pvc_factory(
+            interface=constants.CEPHBLOCKPOOL,
+            project=self.pvcs_cephfs[0].project, size=10,
+            access_modes=access_modes_rbd,
+            status=constants.STATUS_BOUND, num_of_pvc=3, timeout=90
+        )
+
+        pods_cephfs = helpers.create_pods(
+            self.pvcs_cephfs, pod_factory, constants.CEPHFILESYSTEM, 2
+        )
+        pods_rbd = helpers.create_pods(
+            self.pvcs_rbd, pod_factory, constants.CEPHBLOCKPOOL, 2
+        )
+
+        self.pods = pods_cephfs + pods_rbd
+
+        # Set volume mode on PVC objects
+        for pvc_obj in self.pvcs_cephfs+self.pvcs_rbd:
+            pvc_info = pvc_obj.get()
+            setattr(pvc_obj, 'volume_mode', pvc_info['spec']['volumeMode'])
+
+    def expand_and_verify(self, pvc_size_new):
```

Comment:
can this method be used also in tests.manage.pv_services.test_pvc_resize.TestPvcResize.test_pvc_resize?
I think the validation here is relevant also for OCS-325

Reply:
I will move test_pvc_resize to this module in a separate PR.